test/distributed/fsdp/test_fsdp_param_exec_order_policy.py

```diff
@@ -316,6 +342,42 @@ def test_fsdp_flatten_params_exec_order(self, iters: int):
         )
         self.assertTrue(fsdp_model._param_exec_order_state, ParamExecOrderState.INITIALIZED)
 
+    @skip_if_lt_x_gpu(2)
+    @parametrize("mp_config", [
+        default_mp, mp_only_reduce, mp_only_param_and_buf, mp_no_mixed_precision
+    ])
+    def test_fsdp_mixed_precision(self, mp_config: MixedPrecision):
```

Comment:
Curious, did you write this test from scratch?

I am trying to think strategically about how to organize unit tests for the non-recursive wrapping path. It makes sense to have some coverage in `test_fsdp_param_exec_order_policy.py`, but where do the draw the line? (We still will need to add the non-recursive wrapping setting to the existing unit tests. I do not think we will test _every_ feature in this file.)